src/pipelines/extract.py

```diff
@@ -35,7 +35,14 @@ def extract_tickers(db_conn_data):
     """
     load_dotenv()
     if os.getenv("RUN_ENV") == "TESTING":
-        query = "SELECT DISTINCT ticker FROM senator_trades where ticker in ('AAPL','MSFT','BAC','DIS','NFLX') "
+        query = "SELECT DISTINCT ticker FROM senator_trades where ticker in ('AAPL','MSFT','BAC','DIS','NFLX')"
+    
+    elif os.getenv("RUN_ENV") == "INCREMENTAL TESTING 1":
```

Comment:
If we still need this incremental testing, I'd say let's clean this up a bit with a variable:
```
testing_env = os.getenv("RUN_ENV")
if testing_env == "TESTING":
...
```